akka-stream/src/main/scala/akka/stream/impl/io/ByteStringParser.scala

```diff
@@ -177,6 +177,7 @@ import akka.util.ByteString
       throw new IllegalStateException("no initial parser installed: you must use startWith(...)")
   }
 
+  @deprecated("Deprecated for internal usage. Will not be emitted any more.", "2.6.20")
```

Comment:
We need to keep it for a while because e.g. akka-http depends on it currently.

Reply:
Will Akka HTTP still work as expected even though the exception is not wrapped any more?

Reply:
Yes, didn't see any issues.